packages/neo-one-node-data-backup/src/provider/GCloudProvider.ts

```diff
@@ -3,8 +3,9 @@ import { File } from '@google-cloud/storage/build/src/file';
 import { Monitor } from '@neo-one/monitor';
 import * as fs from 'fs-extra';
 import * as path from 'path';
+import * as tar from 'tar';
 import { Environment } from '../types';
-import { extract } from './extract';
+// import { extract } from './extract';
```

Comment:
This should still use `extract`, just replace the implementation with just `tar.extract`

Reply:
Should I just delete all of the stream extract stuff? I was doing this as a temporary disable if we wanted to come back to fixing the `Throttle`. I guess it works tho so maybe we don't even need the throttle now. 🤔

Reply:
Also where does this leave `writeBytesPerSecond`? referring to my *breaking* comment.